tests/console/verify_no_separate_home.pm

```diff
@@ -16,11 +16,14 @@ use testapi;
 sub run {
     select_console 'root-console';
 
-    assert_script_run("! lsblk -n | grep '/home'",
-        fail_message => "Fail!\n
-        Expected: /home is NOT on separate partition/volume.\n
-        Actual: /home is on separate partition/volume."
-    );
+    my $root_device = script_output("findmnt -nrvo SOURCE /");
+    my $home_device = script_output("findmnt -nrvo SOURCE /home");
+    record_info("Source devices", "/ is on ${root_device}, /home is on ${home_device}");
+    if ($root_device ne $home_device) {
```

Comment:
Hi Fabian! Thank you a lot for the PR! 

Just a minor thing, could you please use here `assert_not_equals` method from [Test::Assert](https://metacpan.org/pod/Test::Assert) library, as we are trying to use it everywhere, where asserts are needed. It allows to make all asserts consistent and provide nice stacktrace in case of failure. e.g. similar to `assert_equals`, that we use here:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/d99fc545dfb161c56a105c509ad26b3d99a4283a/tests/console/validate_lvm_raid1.pm#L49

Do not forget to import `use Test::Assert 'assert_not_equals';`

Reply:
That way the `record_info` is also superfluous, nice!